### ISSUE TITLE
It is possible to run other LLM such as LLAMA locally?

### ISSUE THREAD

**Issue by colinljy** [mshumer/gpt-author#12]
Hi, just wondering if it is possible to do something like this but for a local machine maybe with LLAMA or other LLM that is available offline without using an API call?

**Comment by fuleinist** [comment]
yes, it's possible but would require lots of refactor. while it could be possible by connect to another local endpoint service. e.g. [oobabooga](https://github.com/oobabooga/text-generation-webui)

https://www.reddit.com/r/LocalLLaMA/comments/15fxron/best_llama2_model_for_storytelling/
